### ISSUE TITLE
Apple TV app

### ISSUE THREAD

**Issue by cdf1982** [fer0n/Unwatched#8]
Unwatched is undoubtedly the app that had me most excited in years, and it has already replaced the YouTube app on my home screens on iPhone and iPad.

I was wondering if an Apple TV app is planned for the future; no rush, I appreciate this is already a lot of work! 🙏

**Comment by cdf1982** [comment]
I apologize, I just saw this is already answered in the FAQs available in-app.

**Comment by fer0n** [comment]
No worries, you're not the first to ask, and it's helpful to have this as an open issue.

As mentioned in the FAQ, I'm currently focusing on improving the existing versions' performance, but I may consider a tvOS app in the future.

With sync working much better, it's more likely. It will, however, not be anytime soon.

**Comment by cdf1982** [comment]
Sounds great, and your approach is very sensible: one platform at a time is a much healthier way to manage projects!

**Comment by cdf1982** [comment]
Hi.

I just wanted to provide some feedback about the wonderful surprise I found today in TestFlight on my Apple TV 😁

Given the limitation of YT, requiring embedding in a web view that's not made available in tvOS by Apple, I had resigned to a broken workflow: bookmarking on desktop, watching on phone and tablet, and no more YouTube at night on my TV.

The solution that shipped today makes the best app possible given those limitations, making my videos saved for later available on all platforms.
I am perfectly fine and thankful if the best thing possible is having a synced dashboard to launch "strategically" the tvOS YouTube app.

Thanks, big thumbs up from me!
